R/html-tweak.R

```diff
@@ -89,6 +89,7 @@ tweak_tables <- function(html) {
   # Ensure all tables have class="table"
   table <- xml2::xml_find_all(html, ".//table")
   tweak_class_prepend(table, "table")
+  xml2::xml_add_parent(table, "div", style = "display: flex;")
```

Comment:
Can you explain how this works?

Reply:
I've added a comment but am not too sure what a good reference (apart from Julia Evans' CSS zine) would be.

Maybe https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Center_an_element#choices_made

Reply:
Maybe you need to also comment on how the `position`  argument works? I don't understand how flex-box intersects with whatever css kableExtra is generating?

Is this issue really "allow aligning tables in R Markdown"? Or is it actually "allow kableExtra table alignment"? If it's the second maybe we should just not do it.

Reply:
Another way for people to center a table would be to use https://bookdown.org/yihui/rmarkdown-cookbook/kable.html#customize-html-tables and add the style there (or a class and CSS). Even if they did that, they'd still need the parent div of the table to have display grid/flex for being able to center the table.

Reply:
I.e. the CSS would still be something about margin auto, which centers something if the parent div has display flex/grid. I don't know of other ways to horizontally center.